cmd/harmony/main.go

```diff
@@ -379,6 +382,14 @@ func setupNodeAndRun(hc harmonyConfig) {
 
 	nodeconfig.SetPeerID(myHost.GetID())
 
+	if hc.Log.VerbosePrints.Config {
+		utils.Logger().Info().Interface("config", rpc_common.Config{
```

Comment:
@LeoHChen Do you prefer `debug` level over `info` level for this?

Reply:
If the user specifies to output, then I prefer the `info` level for this.

Reply:
agree with @LuttyYang